tools/previous-best/README.md

```diff
@@ -0,0 +1,21 @@
+## Previous year's winners
+
+Given the best solvers from last year, `best_solvers_to_csv.py` search them from the
+corresponding year's registration csv file and produce csv lines
+according to the new year's registration csv format so that the
+best solvers are inserted as non-competitive entries in the
+divisions they won.
+
+Input:
+ - old-registiation: the old registration csv
+ - new-registration: the new registration csv
+ - best-solvers: the best solvers csv with headers for division, and
+   solver name, for each track.
+
+The script only outputs the new lines, they are not automatically added
+to `solvers_divisions.csv`.
```

Comment:
As I mentioned, I think that it'd be best to have this script transform the given `new-registration` csv by adding the entries of the bestof solvers.

Reply:
I didn't want to automate that, since something unexpected could mess up the file.  When touching that file I think some human intervention would be good.  But I'm open to ideas: for instance the changes will have to be committed anyway, and maybe that's a good enough point to do it.  But at the same time that can slip past the eyes of a tired committer...